### PR TITLE
Adding decimal paces on Theme options

### LINKED ISSUE #4284: Themes option, small point
The themes options seems very well done.

I wanted grid lines and they are all there.  But in Panel (More options) the first checkbox is for Size Vertical and horizontal.  There the up-down is set to units of 1.  I think it should be (like the others) in steps of 0.1.

### PR DESCRIPTION
**Pull request by maxwellfundi**
Fixes #4284

### CODE REVIEW

File: instat/ucrELementTextControl.Designer.vb [hunk 1]
```diff
@@ -178,8 +178,8 @@ Partial Class ucrELementTextControl
         '
         'ucrNudsize
         '
-        Me.ucrNudsize.DecimalPlaces = New Decimal(New Integer() {0, 0, 0, 0})
-        Me.ucrNudsize.Increment = New Decimal(New Integer() {1, 0, 0, 0})
+        Me.ucrNudsize.DecimalPlaces = New Decimal(New Integer() {1, 0, 0, 0})
+        Me.ucrNudsize.Increment = New Decimal(New Integer() {1, 0, 0, 65536})
```
Comment (dannyparsons) [inline instat/ucrELementTextControl.Designer.vb]: I'm not sure I understand these changes?

Comment (maxwellfundi) [reply instat/ucrELementTextControl.Designer.vb]: This changes decimal places to 1 from the 0 and the other I changed increment to 0.1. It appears when you change the increment property  of nud 0.1 then it gives that change automatically.

**Comment by shadrackkibet** [comment]
is this ready for review?

**Comment by maxwellfundi** [comment]
yeah it is @shadrackkibet. Have a look at the themes sub dialog to see size now works properly

**Comment by shadrackkibet** [comment]
@dannyparsons i have tested this and it is working fine. This is ready for you to review.,

**Pull request merged**